src/platforms/mirserver/orientationsensor.cpp

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright (C) 2019 UBports
```

Comment:
I guess if copyright is going to be assigned to the foundation, then the full legal name needs to be here, rather than just "UBports" no?

Reply:
That has to be added later, we have not yet received the legal name.